meta-signing-key/scripts/create-user-key-store.sh

```diff
@@ -534,6 +534,7 @@ BOOT_GPG_PASSPHRASE = "$BOOT_GPG_PASS"
 OSTREE_GPGID = "$gpg_key_name"
 OSTREE_GPG_PASSPHRASE = "$GPG_PASS"
 OSTREE_GRUB_PW_FILE = "\${GRUB_PW_FILE}"
+RPM_KEYS_DIR = "$RPM_KEYS_DIR"
```

Comment:
RPM_KEYS_DIR is already defined in this way:
```
RPM_KEYS_DIR = "\${MASTER_KEYS_DIR}/rpm_keys"
```

Please fix your problem from the essential.